common-lib/src/cmr/common/util.clj

```diff
@@ -120,6 +120,22 @@
   [v]
   (when v (str/upper-case v)))
 
+(defn capitalize-words
+  "Capitalize every word in a string"
+  [s]
+  (->> (str/split (str s) #"\b")
+       (map str/capitalize)
+       str/join))
```

Comment:
I don't like adding this function. It's too specific and can only handle the case when the enum list contains the values with the first letter capitalized. You should be able to use the generic match-enum-case to handle it.

Reply:
I disagree about specificity being a bad thing here, but I do agree that `match-enum-case` should be used in case the enum this is being used for is updated to include values that don't include this case